experiments/pam/pam_cortx.c

```diff
@@ -85,7 +86,7 @@ static int login(pam_handle_t *pamh, const char *pUsername, const char *pPasswor
 	}
 
 	/* open the body file */
-	body_file = fopen(body_file_name, "wb");
+	body_file = fopen(body_file_name, "w");
```

Comment:
Will we reuse the file somewhere else?  If not, we should open it with `"w+"` and delete the file before the function returns.  Same for the other `fopen()` call.

Reply:
@tb-s  I am not sure, are we going to reuse the file or not. Task was to store response headers and response body in some file.

Reply:
If we are truncating the file when we open it, chances are we are not reusing it unless this is part of a sequence of commands with a shared dependency on that file.